### DISCUSSION TITLE
GSOC 2024

### DISCUSSION THREAD

**Discussion by SjxSubham** [processing/p5.js-web-editor#3153] [General]
Why does Processing not participate in Gsoc this year 2k24??

**Comment by SableRaf** [comment]
Hi @SjxSubham and thanks for your question! The Processing Foundation applied for GSoC 2024 as we do every year, with what I believe was a [solid project list](https://github.com/processing/Processing-Foundation-GSoC/wiki/Project-Ideas-List-(GSoC-2024)). However, Google did not select us this year. Ultimately, it's their decision, and they do not provide specific reasons. While we were disappointed, it's also great that other projects get the opportunity to participate as well. We will definitely apply again in the future.
